Mathlib/Algebra/Order/GroupWithZero/Canonical.lean

```diff
@@ -243,6 +243,43 @@ lemma pow_lt_pow_succ (ha : 1 < a) : a ^ n < a ^ n.succ := by
 lemma pow_lt_pow_right₀ (ha : 1 < a) (hmn : m < n) : a ^ m < a ^ n := by
   induction' hmn with n _ ih; exacts [pow_lt_pow_succ ha, lt_trans ih (pow_lt_pow_succ ha)]
 
+lemma zpow_lt_zpow_of_lt_one₀ (ha₀ : 0 < a) (ha₁ : a < 1) {m n : ℤ} (hmn : m < n) : a ^ n < a ^ m :=
+  zpow_right_strictAnti₀ ha₀ ha₁ hmn
+
+lemma le_of_zpow_le_zpow₀ (ha : 1 < a) {n m : ℤ} (hmn : a ^ m ≤ a ^ n) :
+    m ≤ n := by
```

Comment:
This is superseded by `zpow_le_zpow_iff_right₀` and should be deleted

Reply:
deleted lemma